### DISCUSSION TITLE
Mac ARM版无法打开

### DISCUSSION THREAD

**Discussion by lunear** [lyswhut/lx-music-desktop#890] [General]
安装在M1 Air上后说文件已损坏，无法打开。重新下载了一遍还是如此。是不兼容还是有问题？是否可以用普通Mac版？

**Comment by Allen1iu99** [comment]
M1 无法安装ARM版，提示文件损坏

**Comment by yu6zhou** [comment]
arm64版本打不开，提示文件损坏；反倒是另一个版本可用。

**Comment by yongcao88** [comment]
一样的问题 用的是最新的Macbook Air M2 显示文件损坏

**Comment by yu6zhou** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
你好，来信已收到，我会尽快给你回复，谢谢。

**Comment by lyswhut** [comment]
https://lxmusic.toside.cn/desktop/faq/install-failed

**Comment by confeitocyan** [comment]
可以不用arm版本，羅賽塔2能轉譯英特爾的版本。

**Reply by yongcao88** [reply]
lyswhut 的link 可以解决arm 版本问题 如果原生arm 的运行肯定是要比x86 模拟的效率高一点
